nextflow_schema.json

```diff
@@ -177,7 +184,6 @@
             },
             "help_text": "The pipeline requires UMI barcodes for identifying unique\ntranscripts. These barcodes are typically read from an index file but sometimes can be provided merged with the start of the R1 or R2 reads. If provided in an additional index file, set the `--index_file` parameter, if provided merged with the R1 or R2 reads, set the `--umi_position` parameter. Specify the UMI barcode length with the `--umi_length` parameter.",
             "required": [
```

Comment:
Actually I should probably make `params.umi` required here right? Alternatively if it would make enough sense to specify no-umi using `params.umi_length` = 0 I can just revert this back to umi_length being a req param.